specification/purview/resource-manager/Microsoft.Purview/stable/2021-12-01/purview.json

```diff
@@ -0,0 +1,2685 @@
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/KafkaConfigurationProperties",
+          "description": "Gets or sets the kafka configuration properties.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "KafkaConfigurationProperties": {
+      "description": "The kafka configuration properties of the event streaming service attached to the Purview account for kafka notifications.",
+      "type": "object",
+      "properties": {
+        "consumerGroup": {
+          "description": "Consumer group for hook event hub.",
+          "type": "string"
+        },
+        "credentials": {
+          "$ref": "#/definitions/Credentials",
+          "description": "Credentials to access event hub."
+        },
+        "eventHubResourceId": {
+          "type": "string"
+        },
+        "eventHubType": {
+          "description": "The event hub type.",
+          "enum": [
+            "Notification",
+            "Hook"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "eventHubType",
+            "modelAsString": true
+          }
+        },
+        "eventStreamingState": {
+          "description": "The state of the event streaming service",
+          "default": "Enabled",
+          "enum": [
+            "Disabled",
+            "Enabled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "eventStreamingState",
+            "modelAsString": true
+          }
+        },
+        "eventStreamingType": {
+          "description": "The event streaming service type",
+          "default": "None",
+          "enum": [
+            "None",
+            "Managed",
+            "Azure"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "eventStreamingType",
+            "modelAsString": true
+          }
+        },
+        "eventHubPartitionId": {
+          "description": "Optional partition Id for notification event hub. If not set, all partitions will be leveraged.",
+          "type": "string"
+        }
+      }
+    },
+    "Credentials": {
+      "description": "Credentials to access the event streaming service attached to the purview account.",
+      "type": "object",
+      "properties": {
+        "identityId": {
+          "description": "Identity identifier for UserAssign type.",
+          "type": "string"
+        },
+        "type": {
+          "description": "Identity Type.",
+          "enum": [
+            "None",
+            "SystemAssigned",
+            "UserAssigned"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "CredentialsType",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "KafkaConfigurationList": {
```

Comment:
Get endpoints for collections of resources must only have the value and nextLink properties in their model.

refs: https://github.com/Azure/azure-openapi-validator/blob/main/docs/get-collection-only-has-value-nextlink.md

Reply:
will remove count

Reply:
Removed count